openwillis/measures/text/speech_attribute.py

```diff
@@ -82,7 +82,6 @@ def is_whisper_transcribe(json_conf):
             return True
     return False
 
-
 def filter_transcribe(json_conf, measures, speaker_label=None):
     """
     ------------------------------------------------------------------------------------------------------
```

Comment:
Update docs to remove phrase in outputs

Reply:
Also are we not doing minimum words per turn in AWS?

Reply:
according to 2.2 method doc, i can not see any variable specific to minimum words per turn

Reply:
what do you mean don't we use it in whisper?